docs/tutorial/voting-tutoral.md

```diff
@@ -0,0 +1,214 @@
+ ```
+
+For our example, this will be
+
+ ```bash
+near create-account yt-voting-example-1.demos.calimero.testnet--masterAccount demos.calimero.testnet --networkId demos-calimero-testnet--nodeUrl https://api.calimero.network/api/v1/shards/demos.calimero.testnet/neard-rpc/  
+ ```
+
+- Replace **<SUB_ACCOUNT_ID>** with your sub account name which can be anything for e.g yt-voting-example
+- Replace **<SHARD_ID>** with your shard name
+
+After, the command has ben ran, you can check in the **Custodial Accounts** section in your console to view the sub account you created.
+
+![](../../static/img/yt-sub-account.png)
+
+## Deploy your NEAR contract
+
+The next step, is to deploy this contract in the private shard. In your cloned repository, navigate to `deploy.calimero.sh`. We will be using the second command which is `near deploy`
+
+```bash
+near deploy \
+  --accountId "" \
+  --initFunction new --initArgs '{"question": "Which blockchain is best?", "options": ["NEAR","Bitcoin"]}' \
+  --wasmFile target/wasm32-unknown-unknown/release/poll.wasm \
+  --nodeUrl "calimero-rpc-node-url" \
+  --networkId "$1-calimero-testnet"
+```
+
+- Replace the _accountID_ to your sub account name. In our example this will be  `yt-voting-example-1.demos.calimero.testnet`
+- Replace _networkID_ to your shard name. In our example this will be `demos-calimero-testnet`
+- For the `calimero-rpc-node-url`, go to the Calimero [Console dashboard](https://app.calimero.network/dashboard) and copy the the near RPC endpoint
+
+For our example, this will be
+
+```bash
+near deploy \
+  --accountId "yt-voting-example-1.demos.calimero.testne" \
+  --initFunction new --initArgs '{"question": "Which blockchain is best?", "options": ["NEAR","Bitcoin"]}' \
+  --wasmFile target/wasm32-unknown-unknown/release/poll.wasm \
+  --nodeUrl "https://api.calimero.network/api/v1/shards/demos-calimero-testnet/neard-rpc/  " \
+  --networkId "demos-calimero-testnet"
+```
+
+If the contract was successfully deployed you can check this on the [Explorer > Transactions](https://app.calimero.network/dashboard/explorer/transactions) on your the console.
+
+![](../../static/img/voting_tranactions.png)
+
+
+## Updating config file and starting up DAPP frontend
+
+To set up the frontend, you'll need to:
+
+- Configure your connection settings in the `calimeroSdk.ts` file.
+
+![](../../static/img/calimero.sdk.png)
+
+:::info
+We recommend using environment variables **.env** to save these settings. Here's a breakdown of what each configuration property does:
+:::
+
+- **NEXT_PUBLIC_CALIMERO_URL**: this is an RPC endpoint used for syncing account and querying shard data and can be found on your Calimero Console dashboard page under endpoints table.
+- **NEXT_PUBLIC_CALIMERO_TOKEN**: auth token created (use the token previously created from the console).
+
+
+- Run the following command to start your http://localhost:3000 
```

Comment:
your `http://localhost:3000` sounds very weird.
Change it to something that describes it's a local version of the dapp